bench/bench.gradle.kts

```diff
@@ -20,19 +20,17 @@ dependencies {
 jmh {
   //include = ["fib_class_java"]
   //include = ["fib_class_constrained1", "fib_class_constrained2"]
-  jmhVersion.set(libs.versions.jmh)
+  jmhVersion = libs.versions.jmh
```

Comment:
In newer versions of Gradle's Kotlin DSL, there is no need to call `.set(...)` on properties; a compiler plugin takes care of the assignment syntax.